examples/byoc_request.ipynb

```diff
@@ -536,7 +513,19 @@
    "cell_type": "code",
    "execution_count": 17,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Failed to download from:\n",
+      "https://services.sentinel-hub.com/api/v1/byoc/collections/b676c6c4-1133-4328-afa7-e9becc5ba590\n",
+      "with HTTPError:\n",
+      "404 Client Error: Not Found for url: https://services.sentinel-hub.com/api/v1/byoc/collections/b676c6c4-1133-4328-afa7-e9becc5ba590\n",
+      "Server response: \"{\"error\":{\"status\":404,\"reason\":\"Not Found\",\"message\":\"Collection b676c6c4-1133-4328-afa7-e9becc5ba590 not found.\",\"code\":\"COMMON_NOT_FOUND\"}}\"\n"
+     ]
+    }
+   ],
```

Comment:
This output shouldn't be here. Does this work otherwise? Perhaps you can just clear the output cell?

Reply:
Thanks for reminding. I clear the output.